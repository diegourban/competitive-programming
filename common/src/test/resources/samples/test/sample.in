some content
1
5.55
8712367891236
y