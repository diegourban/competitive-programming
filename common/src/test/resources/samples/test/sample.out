1
o
3.14
pi
